### DISCUSSION TITLE
How to override super variable？

### DISCUSSION THREAD

**Discussion by Charliego3** [progrium/darwinkit#69] [Q&A] [answered]
For example: `Splitview.dividerThickness`
```swift
class SplistView: NSSplitView {
    override var dividerThickness: CGFloat {
       return 0
    }
}
```
![屏幕快照 2021-09-27 下午9 15 58](https://user-images.githubusercontent.com/26761592/134916067-13f2ec6e-1b79-4276-8e86-06e66179849e.png)
 I want to remove the SplitView divider.  
How can this be achieved？

**Comment by progrium** [comment]
Can you just set it in some initialization code?

**Comment by progrium** [comment, marked as answer]
Based on the Objective-C documentation (which is what you should use when working with MacDriver):
https://developer.apple.com/documentation/appkit/nssplitview/1455257-dividerthickness?language=objc

This is a property, which are just declaring getter and/or setter methods. You would be able to override this implementing a "dividerThickness" method on a subclass.

**Reply by Charliego3** [reply]
Thanks a lot, I will try it later.

**Reply by progrium** [reply]
Btw we incorporated this into our subclass example:
https://github.com/progrium/macdriver/blob/main/macos/_examples/subclass/main.go
